root/edit/details/AddRelationship.js

```diff
@@ -50,7 +50,7 @@ const AddRelationship = ({edit}: Props): React.MixedElement => {
           <td>{edit.display_data.relationship.linkOrder}</td>
         </tr>
       ) : null}
-      {edit.display_data.unknown_attributes ? (
+      {edit.display_data.unknown_attributes /*:: === true */ ? (
```

Comment:
It looks like `unknown_attributes` is always outputted by `Edit::Relationship::Create::build_display_data` - maybe this should be non-optional in the type instead?